addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/AutelisBindingConstants.java

```diff
@@ -24,8 +24,11 @@
 
     public static final String BINDING_ID = "autelis";
 
-    // List of all Thing Type UIDs
+    // poolcontrol is here for backwards compatibility before we had separate things for jandy and pentair
```

Comment:
Did you confirm this actually works for existing paper UI items and existing textual config items?

Reply:
I will double check to make sure, and make changes if it does not.

Reply:
> > Did you confirm this actually works for existing paper UI items and existing textual config items?

> I will double check to make sure, and make changes if it does not.

Where you able to validate this?